package.json

```diff
@@ -22,13 +22,26 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6"
   },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "<rootDir>src/setupTests.js"
+    ],
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx}",
+      "!src/index.js",
+      "!src/setupTests.js"
+    ]
+  },
   "devDependencies": {
     "@babel/core": "^7.4.5",
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
     "babel-eslint": "^10.0.2",
     "babel-loader": "^8.0.6",
     "css-loader": "^3.0.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
```

Comment:
The two dependencies(enzyme and enzyme-adapter-react-16) should be listed in dependencies instead of devDependencies.

Reply:
Okay @reifred, let me work on that please. Thanks